### ISSUE TITLE
DL same version of Mayan docs as installed version

### ISSUE THREAD

**Issue by JedMeister** [DocCyblade/tkl-mayan-edms#40]
Just saw this:

https://github.com/DocCyblade/tkl-mayan-edms/blob/development/conf.d/main#L112

````
DEMO_FILE_URL2=http://readthedocs.org/projects/mayan/downloads/pdf/v2.1.3/
````

Probably should be:

````
DEMO_FILE_URL2=http://readthedocs.org/projects/mayan/downloads/pdf/${MAYAN_APP_VER}/
````

**Comment by DocCyblade** [comment]
Since my idea of locking the build to a specific mayan release, and just use the latest at build time, ill just use the latest.

**Comment by DocCyblade** [comment]
Fixed with commit dce8dc1
